reframe/frontend/printer.py

```diff
@@ -20,6 +20,19 @@ def __init__(self):
         self.line_width = 78
         self.status_width = 10
 
+    def set_verbosity(self, number):
+        if number:
+            for hdlr in self.stream_handlers():
+                for l in range(0,number):
+                    loglevel = hdlr.level
+                    if loglevel == logging.INFO:
```

Comment:
I think you can avoid this set of `if`s if you do the following:

```python
def inc_verbosity(self, num_steps):
    log_levels = sorted(_log_level_names.keys())[1:] # [1:] to exclude level NOTSET
    for h in self.stream_handlers:
        level_idx = log_levels.index(h.level)
        try:
            new_level = log_levels[level_idx - num_steps]
        except IndexError:
            new_level = 0

        h.setLevel(new_level)
```

Reply:
I also believe `inc_verbosity()` is better than `set_verbosity()`, because you can only increase the verbosity.